execution_engine_testing/tests/benches/auction_bench.rs

```diff
@@ -0,0 +1,297 @@
+const TIMESTAMP_INCREMENT_MILLIS: u64 = 30_000;
+
+fn run_genesis_and_create_initial_accounts(
+    data_dir: &Path,
+    validator_keys: &[PublicKey],
+    delegator_accounts: Vec<AccountHash>,
+) -> LmdbWasmTestBuilder {
+    let engine_config = EngineConfig::default();
+    let mut builder = LmdbWasmTestBuilder::new_with_config(data_dir, engine_config);
+
+    let mut genesis_accounts = vec![
+        GenesisAccount::account(
+            DEFAULT_ACCOUNT_PUBLIC_KEY.clone(),
+            Motes::new(U512::MAX), // all the monies
+            Some(GenesisValidator::new(
+                Motes::new(U512::from(VALIDATOR_BID_AMOUNT)),
+                DELEGATION_RATE,
+            )),
+            // None,
+        ),
+        GenesisAccount::account(
+            DEFAULT_PROPOSER_PUBLIC_KEY.clone(),
+            Motes::new(DEFAULT_ACCOUNT_INITIAL_BALANCE.into()),
+            None,
+        ),
+    ];
+    for validator in validator_keys {
+        genesis_accounts.push(GenesisAccount::account(
+            validator.clone(),
+            Motes::new(U512::MAX), // all the monies
+            Some(GenesisValidator::new(
+                Motes::new(U512::from(VALIDATOR_BID_AMOUNT)),
+                DELEGATION_RATE,
+            )),
+        ))
+    }
+    let run_genesis_request =
+        create_run_genesis_request(validator_keys.len() as u32 + 2, genesis_accounts);
+    builder.run_genesis(&run_genesis_request);
+
+    {
+        // Setup the system account with enough cspr
+        let transfer = ExecuteRequestBuilder::transfer(
+            *DEFAULT_ACCOUNT_ADDR,
+            runtime_args! {
+                    ARG_TARGET => *SYSTEM_ADDR,
+                    ARG_AMOUNT => MINIMUM_ACCOUNT_CREATION_BALANCE,
+                    ARG_ID => <Option<u64>>::None,
+            },
+        )
+        .build();
+        builder.exec(transfer);
+        builder.expect_success().commit();
+    }
+
+    for delegator_account in delegator_accounts {
+        let transfer = ExecuteRequestBuilder::transfer(
+            *DEFAULT_ACCOUNT_ADDR,
+            runtime_args! {
+                    ARG_TARGET => delegator_account,
+                    ARG_AMOUNT => DELEGATOR_INITIAL_BALANCE,
+                    ARG_ID => <Option<u64>>::None,
+            },
+        )
+        .build();
+        builder.exec(transfer);
+        builder.expect_success().commit();
+    }
+    builder
+}
+
+fn create_run_genesis_request(
+    validator_slots: u32,
+    genesis_accounts: Vec<GenesisAccount>,
+) -> RunGenesisRequest {
+    let exec_config = {
+        let wasm_config = *DEFAULT_WASM_CONFIG;
+        let system_config = *DEFAULT_SYSTEM_CONFIG;
+        let auction_delay = DEFAULT_AUCTION_DELAY;
+        let locked_funds_period_millis = DEFAULT_LOCKED_FUNDS_PERIOD_MILLIS;
+        let round_seigniorage_rate = DEFAULT_ROUND_SEIGNIORAGE_RATE;
+        let unbonding_delay = DEFAULT_UNBONDING_DELAY;
+        let genesis_timestamp_millis = DEFAULT_GENESIS_TIMESTAMP_MILLIS;
```

Comment:
We don't need these, right? We can pass the constants to `ExecConfig::new`.

Reply:
Artifacts of inlining the method, I'll clean up the temporaries.